frontend/src/app/app-routing.module.ts

```diff
@@ -24,7 +24,7 @@ const routes: Routes = [
     canActivate: [AuthGuard, ProjectGuard],
   },
   {
-    path: 'project/:projectID/board/:boardID',
+    path: 'project/:projectID/:boardType/:boardID',
```

Comment:
Small change: can we use query parameters in the url instead of the changing the path? I.e. project/i28djwj/board/isn274djs?embedded=true

You can check out this page to see how that's done with Angular routing:

How to read query Params:
https://www.angularjswiki.com/angular/get-query-parameters-in-angular/

Reply:
@Ammar-T Good point, let's try that

Reply:
Sounds good, I've added the query params